apps/web/components/apps/App.tsx

```diff
@@ -77,14 +63,11 @@ const Component = ({
   }).format(price);
 
   const [existingCredentials, setExistingCredentials] = useState<number[]>([]);
-  const [showDisconnectIntegration, setShowDisconnectIntegration] = useState(false);
-  const appDbQuery = trpc.viewer.appCredentialsByType.useQuery(
+  const appCredentials = trpc.viewer.appCredentialsByType.useQuery(
     { appType: type },
     {
-      onSettled(data) {
-        const credentialsCount = data?.credentials.length || 0
-        setShowDisconnectIntegration(data?.userAdminTeams.length ? credentialsCount >= data?.userAdminTeams.length : credentialsCount > 0);
-        setExistingCredentials(data?.credentials.map(credential => credential.id) || []);
```

Comment:
Needs a comment on why this code was modified.

Reply:
i think this was just prettier and eslint doing its thing

Reply:
@hariombalhara ive reverted all changes to app.tsx.